SketchUp/Sketchup/AttributeDictionary.rb

```diff
@@ -271,7 +271,7 @@ def size
   #   attrdict = model.attribute_dictionaries['test_dict']
   #   values = attrdict.values
   #
-  # @return keys - an array of keys within the attribute dictionary
+  # @return [Array<Object, nil>] an array of values within the attribute dictionary
```

Comment:
For type inference yes. For documentation I thought it's better to have `nil` explicit to emphasis that a key may not be found. I will correct 'values' instead of keys.